src/main/java/io/appform/memq/ExecutorServiceProvider.java

```diff
@@ -0,0 +1,9 @@
+package io.appform.memq;
+
+import java.util.concurrent.ExecutorService;
+
+public interface ExecutorServiceProvider {
```

Comment:
Remove this and take a `Supplier<ExecutorService>` instead

Reply:
Done